aspnetcore/tutorials/min-web-api.md

```diff
@@ -0,0 +1,341 @@
+---
+title: "Tutorial: Create a minimal web API with ASP.NET Core"
+author: rick-anderson
+description: Learn how to build a minimal web API with ASP.NET Core.
+ms.author: riande
+ms.date: 8/24/2021
+monikerRange: '>= aspnetcore-6.0'
+no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR, Models]
+uid: tutorials/min-web-api
+---
+
+<!-- test with windows sandbox -->
+# Tutorial: Create a web minimal API with ASP.NET Core
+
+By [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+Minimal APIs are architected to create REST APIs with the minimal dependencies. They are ideal for microservice and app that don't want to include only the minimum files, features, and dependencies in ASP.NET Core.
+
+This tutorial teaches the basics of building a miminal web API with ASP.NET Core. For a tutorial on creating a web API project that contains more features, see [Create a web API](xref:tutorials/first-web-api)
+
+## Overview
+
+This tutorial creates the following API:
+
+|API | Description | Request body | Response body |
+|--- | ---- | ---- | ---- |
+|`GET /api` | Browser test, "Hello World" | None | `Hello World!`|
+|`GET /TodoItems` | Get all to-do items | None | Array of to-do items|
```

Comment:
This might be a style thing but lowercase URLs are more normal. /todoitems

Reply:
> This might be a style thing but lowercase URLs are more normal. `/todoitems`

@davidfowl  I was copying the scaffolder which creates `/api/TodoItems`

Two questions:

* Should I append `/api` to be consistent with the scaffolder and [Web API tutorial](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-6.0&branch=pr-en-us-22941&tabs=visual-studio)
* Should I open a scaffolding issue and request they generate lowercase endpoints?